xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemesCode/IconPicker.xml

```diff
@@ -243,8 +243,7 @@ require(['jquery', 'xwiki-icon-picker'], function($) {
       selectedIconName = currentInput[0].value.replace(currentInput.data('xwikiIconPickerSettings').prefix, '');
     }
     // For each icon
-    for (var i=0; i &lt; iconList.length; ++i) {
-      let icon = iconList[i];
+    iconList.forEach(icon => {
```

Comment:
You need a closing ``)`` at the end (I couldn't add it as a code suggestion :) ).